arches_rdm/etl_modules/migrate_rdm.py

```diff
@@ -0,0 +1,195 @@
+            for value in concept.value_set.all():
+                # scheme_to_load["legacyid"] = value.concept_id
+                scheme_to_load["resourceinstanceid"] = value.valueid # use old valueid as new resourceinstanceid
+
+                name = {}
+                name["name_content"] = value.value
+                # name["name_language"] = value.language
+                # name["name_type"] = value.type
+                scheme_to_load["tile_data"].append({"name": name})
+            schemes.append(scheme_to_load)
+        self.populate_staging_table(cursor, schemes, nodegroup_lookup, node_lookup)       
+
+    def etl_concepts(self, cursor, nodegroup_lookup, node_lookup):
+        concepts = []
+        for concept in models.Concept.objects.filter(nodetype="Concept").prefetch_related("value_set"):
+            concept_to_load = {"type": "Concept", "tile_data": []}
+            for value in concept.value_set.all():
+                # concept_to_load["legacyid"] = value.concept_id
+                concept_to_load["resourceinstanceid"] = value.valueid # use old valueid as new resourceinstanceid
+
+                name = {}
+                name["name_content"] = value.value
+                # name["name_language"] = value.language
+                # name["name_type"] = value.type
+                concept_to_load["tile_data"].append({"name": name})
+            concepts.append(concept_to_load)
+        self.populate_staging_table(cursor, concepts, nodegroup_lookup, node_lookup)
+
+
+    def populate_staging_table(self, cursor, concepts_to_load, nodegroup_lookup, node_lookup):
+        tiles_to_load = []
+        for concept_to_load in concepts_to_load:
+            for mock_tile in concept_to_load["tile_data"]:
+                nodegroup_alias = next(iter(mock_tile.keys()), None)
+                nodegroup_id = node_lookup[nodegroup_alias]["nodeid"]
+                nodegroup_depth = nodegroup_lookup[nodegroup_id]["depth"]
+                tile_id = uuid.uuid4()
+                parent_tile_id = None
+                tile_value_json, passes_validation = self.create_tile_value(cursor, mock_tile, nodegroup_alias, nodegroup_lookup, node_lookup)
+                operation = "insert"
+                tiles_to_load.append(LoadStaging(
+                    load_event=LoadEvent(self.loadid),
+                    nodegroup=NodeGroup(nodegroup_id),
+                    resourceid=concept_to_load["resourceinstanceid"],
+                    tileid=tile_id,
+                    parenttileid=parent_tile_id,
+                    value=tile_value_json,
+                    nodegroup_depth=nodegroup_depth,
+                    source_description="{0}: {1}".format(concept_to_load["type"], nodegroup_alias),  # source_description
+                    passes_validation=passes_validation,
+                    operation=operation,
+                ))
+        staged_tiles = LoadStaging.objects.bulk_create(tiles_to_load)
```

Comment:
Nice! This bulk create is working well. My jobs failed after this point, but this part seems to work quickly enough that you don't need celery.